docs/core/install/linux-fedora.md

```diff
@@ -24,18 +24,16 @@ The following table is a list of currently supported .NET releases and the versi
 
 | Fedora | .NET      |
 |--------|-----------|
-| 37     | 6         |
-| 36     | 6         |
-| 35     | 6         |
+| 37     | 7         |
+| 36     | 7         |
+| 35     | 7         |
 
-> [!IMPORTANT]
-> .NET 7 isn't yet ready for Fedora. This article will be updated when it's available.
 
 [!INCLUDE [versions-not-supported](includes/versions-not-supported.md)]
 
-## Install .NET 6
+## Install .NET 7
 
-[!INCLUDE [linux-dnf-install-60](includes/linux-install-60-dnf.md)]
+[!INCLUDE [linux-dnf-install-70](includes/linux-install-70-dnf.md)]
```

Comment:
Why are you removing .NET 6? It's still supported, correct?

Reply:
Sorry I made this merge request when it said that dotnet 7 wasn't supported but it is now updated.